cms/djangoapps/contentstore/utils.py

```diff
@@ -192,30 +192,26 @@ def get_lms_link_for_item(location, preview=False):
     """
     assert isinstance(location, UsageKey)
 
-    # checks LMS_BASE value in site configuration for the given course_org_filter(org)
-    # if not found returns settings.LMS_BASE
     lms_base = SiteConfiguration.get_value_for_org(
         location.org,
         "LMS_BASE",
         settings.LMS_BASE
     )
+    params = ''
 
     if lms_base is None:
         return None
 
     if preview:
-        # checks PREVIEW_LMS_BASE value in site configuration for the given course_org_filter(org)
-        # if not found returns settings.FEATURES.get('PREVIEW_LMS_BASE')
-        lms_base = SiteConfiguration.get_value_for_org(
-            location.org,
-            "PREVIEW_LMS_BASE",
-            settings.FEATURES.get('PREVIEW_LMS_BASE')
-        )
+        # checks LMS_BASE value in site configuration for the given course_org_filter(org)
+        # if not found returns settings.LMS_BASE
+        params = '?preview=1'
 
-    return "//{lms_base}/courses/{course_key}/jump_to/{location}".format(
+    return "//{lms_base}/courses/{course_key}/jump_to/{location}{params}".format(
         lms_base=lms_base,
         course_key=str(location.course_key),
         location=str(location),
+        params=params,
     )
```

Comment:
Nit: I know this follows existing patterns but probably better to use a URL builder here.